weave/trace/op.py

```diff
@@ -195,7 +196,9 @@ def _is_unbound_method(func: Callable) -> bool:
     return bool(is_method)
 
 
-def _create_call(func: Op, *args: Any, **kwargs: Any) -> "Call":
+def _create_call(
+    func: Op, *args: Any, wvkw: Optional[dict] = None, **kwargs: Any
```

Comment:
`wvkw -> weave_kwargs`

Reply:
Also this needs to be explicitly typed so people know what the options are

Reply:
done